### PR TITLE
magento version one staging

### PR DESCRIPTION
**Pull request by jonmediajel**

### CODE REVIEW

File: src/shared/environment-data-sources/magento.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { EnvironmentEvents, TransactionCartItem } from "../types";
+import { xhrResponseSource } from "../sources/xhr-response-source";
+
+const magentoDataSource = ({ transactionEvent }: Partial<EnvironmentEvents>) => {
+  xhrResponseSource((xhr) => {
+    try {
+      const getData = JSON.parse(JSON.stringify(JSON.parse(xhr.responseText)));
```
Comment (pacholoamit) [inline src/shared/environment-data-sources/magento.ts:7]: why do we parse then stringify then parse again?

File: src/v2/imports/carts/magento.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    } catch (error) {}
+  });
+  try {
+    if (window.location.pathname.includes("/checkout/onepage/success/")) {
+      setTimeout(() => {
+        const storedData = sessionStorage.getItem("pixelData");
+        const retrievedObject = JSON.parse(storedData);
+        const productsList = retrievedObject && retrievedObject.items;
+
+        const checkoutSuccessElement = document.querySelector(".checkout-success");
```
Comment (pacholoamit) [inline src/v2/imports/carts/magento.ts:24]: This entire section is incorrect. Lets sync up regarding what the v2 is for

**Review by pacholoamit — CHANGES_REQUESTED** [review]
Hello, please see feedback. Thanks!

**Comment by jonmediajel** [comment]
Thanks for the review @pacholoamit. Feedbacks are now fixed

**Comment by pacholoamit** [comment]
> Thanks for the review @pacholoamit. Feedbacks are now fixed

Thanks

File: src/v2/imports/carts/magento.ts
Comment (pacholoamit) [inline src/v2/imports/carts/magento.ts:1]: you should be able to use the `magentoDataSource` here... Similar to what you did on `src/v1/imports/carts/magento.ts`

**Comment by jonmediajel** [comment]
all goods now @pacholoamit , thanks

**Review by pacholoamit — APPROVED** [review]
Beautiful

**Pull request merged**